README.md

```diff
@@ -122,28 +125,117 @@ module.exports = [{
 }];
 ```
 
-### Tags
-Tags tell the test suites which tests should be run on your issuer, verifier,
-and status lists.
+To run the tests only against the localhost implementation, update the test
+suite to filter implementations using the specified tag in your config file.
 
-* `vc-api` - This tag will run the [vc-api-issuer tests](https://github.com/w3c-ccg/vc-api-issuer-test-suite) on your issuer and the [vc-api-verifier tests](https://github.com/w3c-ccg/vc-api-verifier-test-suite) on your verifier.
+For instance, if your `.vcApiTestImplementationsConfig.cjs` looks like above
+in the `vc-api-issuer-test-suite`, you can adjust the
+[tag](https://github.com/w3c-ccg/vc-api-issuer-test-suite/blob/main/tests/10-issuer.js#L14)
+to filter the implementation by `localhost` rather than `vc-api` and
+then run the tests.
 
-* `Ed25519Signature2020` - This tag will run the [Ed25519 tests](https://github.com/w3c/vc-di-ed25519signature2020-test-suite) on your issuer and/or verifier.
+### Opting into a Test Suite
 
-* `StatusList2021` - This tag will run the [Status List 2021 tests](https://github.com/w3c-ccg/status-list-2021-test-suite) on your issuer and/or verifier.
+Please Note:
 
-* `did-key` - This tag will run the [DID Key Test Suite](https://github.com/w3c-ccg/did-key-test-suite) on your DID resolver endpoint.
+1. Tags serve as identifiers to determine which test suites to run on your
+issuers and verifiers. Only the first issuer/verifier found with a specific
```

Comment:
this is actually test specific in actuality the test suite will see all endpoints marked with a tag and there could be a test suite that wants to run on multiple endpoints with the same tag. The later test does not exist yet, but could happen such as picking on vc-api compatible endpoint at random for an interop test.

Reply:
@aljones15 I am having trouble coming up with an alternative, could you please provide what you would prefer to see here.

Reply:
1. Tags serve as identifiers to determine which test suites to run on your
```suggestion
issuers and verifiers. Only the first issuer/verifier found with a specific
```
issuers and verifiers. So far the first issuer/verifier found with a specific

Reply:
@aljones15 Updated the wording to not include `Only` [03e6fe0](https://github.com/w3c-ccg/vc-test-suite-implementations/pull/91/commits/03e6fe00d618bf23edd8e0d681d7db76e6be37bb).